erpnext/schools/doctype/assessment_result_tool/assessment_result_tool.js

```diff
@@ -7,17 +7,19 @@ cur_frm.add_fetch("assessment_plan", "student_batch", "student_batch");
 frappe.ui.form.on('Assessment Result Tool', {
 
 	onload: function(frm) {
+		frm.trigger("refresh");
+	},
+
+	refresh: function(frm) {
 		if (frappe.route_options) {
 			frm.doc.assessment_plan = frappe.route_options.assessment_plan;
+			frm.trigger("assessment_plan");
```

Comment:
Actually if we are using the back button, it is rendering the table from the cache, that's why I have to forcefully trigger it to render the table again